src/network/networkd-network.c

```diff
@@ -1454,6 +1457,40 @@ int config_parse_dhcp_route_table(const char *unit,
         return 0;
 }
 
+int config_parse_ipv6_accept_ra_route_table(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Network *network = data;
+        uint32_t rt;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        r = safe_atou32(rvalue, &rt);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r,
+                           "Unable to read RouteTable, ignoring assignment: %s", rvalue);
+                return 0;
+        }
+
+        network->ipv6_accept_ra_route_table = rt;
+        network->ipv6_accept_ra_route_table_set = true;
+
+        return 0;
+}
+
```

Comment:
I think ```config_parse_ipv6_accept_ra_route_table``` and ```config_parse_dhcp_route_table``` can be merged depending on the section we set the value.

Reply:
Nice!